imports/plugins/core/versions/server/migrations/25_update_catalog_schema.js

```diff
@@ -0,0 +1,46 @@
+import { Migrations } from "meteor/percolate:migrations";
+import _ from "lodash";
+import { Catalog, Shops } from "/lib/collections";
+import convertCatalogItem from "../util/convertCatalogItem";
+
+// Do this migration in batches of 200 to avoid memory issues
+const LIMIT = 200;
+
+function getShopsForItems(items) {
+  const uniqueShopIds = _.uniq(items.map(({ shopId }) => shopId));
+
+  return Shops.find(
+    {
+      _id: { $in: uniqueShopIds }
+    },
+    {
+      fields: {
+        currencies: 1,
+        currency: 1
+      }
+    }
+  ).fetch();
+}
+
+Migrations.add({
+  version: 25,
+  up() {
+    // We moved everything to a `product` object, so checking for the existence of that
+    // seems like a good way to find everything we haven't converted yet.
+    let items;
+
+    do {
+      items = Catalog.find({ product: { $exists: false } }, { limit: LIMIT }).fetch();
```

Comment:
I was on the fence about whether a migration should change updatedAt anyway. Thoughts? It is technically updating the doc, but it might also be misleading for all of the updatedAt to update since they were more "rearranged" than "updated"